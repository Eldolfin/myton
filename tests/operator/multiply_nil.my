print(None * 3) # error
