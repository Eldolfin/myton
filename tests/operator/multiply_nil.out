[31merror[error]: unsupported operand type(s) for *: 'NoneType' and 'number'
----- Traceback -----
  â”Œâ”€ <unknown>:1:12
  | 
1 | print(None * 3) # error
  |             ^
[0m